Cargo.toml

```diff
@@ -110,8 +110,8 @@ lazy_static           = "1.4"
 libipld               = { version = "0.14", default-features = false, features = ["dag-cbor", "dag-json", "derive"] }
 libipld-core          = "0.14"
 libipld-macro         = "0.14"
-libp2p                = { version = "0.46", default-features = false }
-libp2p-bitswap        = { git = "https://github.com/ChainSafe/libp2p-bitswap", rev = "47a5722767043ec1495bce2bffe02d4b1f5d87e0" }
+libp2p                = { version = "0.49", default-features = false }
+libp2p-bitswap        = { git = "https://github.com/hanabi1224/libp2p-bitswap", branch = "forest", features = ["compat"] }
```

Comment:
Should we push your branch to the ChainSafe bitswap repository?

Reply:
That will be great, I don't think I have write access, would u push it and let me know?

Reply:
I can merge a PR to that repository. Let's do it after this PR has been merged.

Reply:
Sounds good